agent/src/agent/pipeline/manager.py

```diff
@@ -123,7 +123,6 @@ def update(pipeline_: Pipeline):
     if pipeline_.uses_schema:
         _update_schema(pipeline_)
     sdc_client.update(pipeline_)
-    pipeline.repository.save(pipeline_)
```

Comment:
This line deleted, because `repository.save(pipeline_)` will be performed in SessionManager's context `__exit__`